rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -488,25 +489,50 @@ private void saveNewUTXOs(BtcTransaction btcTx) throws IOException {
      */
     public void releaseBtc(Transaction rskTx) throws IOException {
 
-        //as we can't send btc from contracts we want to send them back to the sender
+        Coin value = rskTx.getValue().toBitcoin();
+        final RskAddress senderAddress = rskTx.getSender();
+        final String senderAddressStr = senderAddress.toHexString();
```

Comment:
Any reason to calculate this value in this method at this point? Apparently, it could be calculated inside the new private method `refundAndEmitEvent`

Reply:
You are right, missed in the refactor.